joomla/joomla-cms
# Implementing com_contact frontend editing

This PR adds frontend editing to com_contact.

### Summary of Changes
Adding form view and accompagning controller and model to com_contact frontend


### Testing Instructions
* Test adding and editing contacts from frontend. If the user is authorised, he should see an edit link in the category view and contact view. The category view additionaly has a "New" button.
The featured view doesn't have any such links.
There is a menuitem to point to create new contacts.

* Test that data doesn't get lost when saving (especially data not available to edit in frontend vs backend).

* Test especially also ACL. JSST should look over it for possible security issues.

### Expected result
Works


### Actual result
Frontend editing not possible.


### Documentation Changes Required
A new help page will be needed. Possibly also other doc pages.
